src/app/core/session/login/login.component.html

```diff
@@ -16,7 +16,7 @@
   -->
 
 <mat-card class="login-panel centered mat-elevation-z2">
-  <mat-card-title i18n="Sign in title"> Please Sign In </mat-card-title>
+  <mat-card-title i18n="Sign in title"> Please Sign In</mat-card-title>
```

Comment:
somewhat unrelated but can we just trim spaces from the username as part of this PR as well? We recently had user support cases struggling there again.